src/pal/src/loader/module.cpp

```diff
@@ -676,7 +676,7 @@ PAL_RegisterModule(
 {
     HINSTANCE hinstance = nullptr;
 
-    int err = PAL_InitializeDLL();
+    int err = PAL_InitializeDLLWithoutSEH();
```

Comment:
First, I backed off suggesting we remove calling DllMain during LoadLibrary. Too many dlls depend on this. And now maybe we should back off on removing PAL_RegisterModule/UnregisterModule because those VS dlls may use the module handle and if we remove these exports, the RC2 dbgshim won’t be able to load the newer dbi dlls because the exports would be missing.

So we need to go back to JanV’s suggestion of chaining SIGRTMIN and making that work which would mean if there are no threads in a particular PAL that need to be woken up or terminated, then the signal needs to be chain to the next pal.